SharedAssemblyInfo.cs

```diff
@@ -1,13 +1,13 @@
-using System.Reflection;
+﻿using System.Reflection;
 using System.Runtime.InteropServices;
 
 [assembly: AssemblyDescription("NHibernate.Spatial is a library of spatial extensions for NHibernate, and allows you to connect NHibernate to a spatially enabled database and manipulate geometries in HQL or in .NET code using NetTopologySuite, providing you with a fully integrated GIS programming experience.")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("NHibernate.Spatial Community")]
 [assembly: AssemblyProduct("NHibernate.Spatial")]
-[assembly: AssemblyCopyright("Copyright � 2009-2017 Ricardo Stuven, Surya Pratap, Andreas Ravnestad, Peet Whitaker")]
+[assembly: AssemblyCopyright("Copyright © 2009-2017 Ricardo Stuven, Surya Pratap, Andreas Ravnestad, Peet Whitaker")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: ComVisible(false)]
-[assembly: AssemblyVersion("4.1.1.4001")]
-[assembly: AssemblyFileVersion("4.1.1.4001")]
\ No newline at end of file
+[assembly: AssemblyVersion("5.0.0")]
+[assembly: AssemblyFileVersion("5.0.0")]
```

Comment:
> Oh, and I guess you might want to update the NHibernate.Spatial shared version number to 5, to match NHibernate

And here in `SharedAssemblyInfo.cs`. Is there somewhere else I would have missed?

Note that NHibernate has dropped the .4000/GA suffix, and I have done same here.